llvm/test/CodeGen/AIE/aie2/schedule/swp/disable.ll

```diff
@@ -0,0 +1,119 @@
+  %s.05 = phi i32 [ %or, %for.body ], [ 0, %entry ]
+  %p.addr.04 = phi ptr [ %incdec.ptr, %for.body ], [ %p, %entry ]
+  %incdec.ptr = getelementptr inbounds i32, ptr %p.addr.04, i20 1
+  %cst = xor i32 %i.06, 7
+  %0 = load i32, ptr %p.addr.04, align 4, !tbaa !2
+  %xor = xor i32 %0, %cst
+  %or = or i32 %xor, %s.05
+  %inc = add nuw nsw i32 %i.06, 1
+  %exitcond.not = icmp eq i32 %inc, %n
+  br i1 %exitcond.not, label %for.cond.cleanup, label %for.body
+
+for.cond.cleanup:                                 ; preds = %for.body, %entry
+  ret i32 %or
+
+}
+
+define dso_local i32 @pragma(ptr nocapture readonly %p, i32 noundef %n) {
+  ; CHECK-LABEL: name: pragma
+  ; CHECK: bb.0.entry (align 16):
+  ; CHECK-NEXT:   successors: %bb.1(0x80000000)
+  ; CHECK-NEXT:   liveins: $p0, $r1
+  ; CHECK-NEXT: {{  $}}
+  ; CHECK-NEXT:   renamable $r2 = MOV_RLC_imm10_pseudo 7
+  ; CHECK-NEXT:   renamable $r3 = MOV_RLC_imm10_pseudo 0
+  ; CHECK-NEXT:   renamable $r0 = MOV_RLC_imm10_pseudo 0
+  ; CHECK-NEXT: {{  $}}
+  ; CHECK-NEXT: bb.1.for.body (align 16):
+  ; CHECK-NEXT:   successors: %bb.2(0x04000000), %bb.1(0x7c000000)
+  ; CHECK-NEXT:   liveins: $p0, $r0, $r1, $r2, $r3
+  ; CHECK-NEXT: {{  $}}
+  ; CHECK-NEXT:   renamable $r4 = XOR renamable $r3, renamable $r2
+  ; CHECK-NEXT:   renamable $r5, renamable $p0 = LDA_dms_lda_pstm_nrm_imm killed renamable $p0, 4 :: (load (s32) from %ir.p.addr.04, !tbaa !0)
+  ; CHECK-NEXT:   renamable $r4 = XOR killed renamable $r5, killed renamable $r4
+  ; CHECK-NEXT:   renamable $r0 = OR killed renamable $r4, killed renamable $r0
+  ; CHECK-NEXT:   renamable $r3 = nuw nsw ADD_add_r_ri killed renamable $r3, 1, implicit-def dead $srcarry
+  ; CHECK-NEXT:   renamable $r4 = NE renamable $r1, renamable $r3
+  ; CHECK-NEXT:   JNZ killed renamable $r4, %bb.1
+  ; CHECK-NEXT:   DelayedSchedBarrier
+  ; CHECK-NEXT: {{  $}}
+  ; CHECK-NEXT: bb.2.for.cond.cleanup:
+  ; CHECK-NEXT:   liveins: $r0
+  ; CHECK-NEXT: {{  $}}
+  ; CHECK-NEXT:   RET implicit $lr
+  ; CHECK-NEXT:   DelayedSchedBarrier implicit $r0
+entry:
+  br label %for.body
+
+for.body:                                         ; preds = %entry, %for.body
+  %i.06 = phi i32 [ %inc, %for.body ], [ 0, %entry ]
+  %s.05 = phi i32 [ %or, %for.body ], [ 0, %entry ]
+  %p.addr.04 = phi ptr [ %incdec.ptr, %for.body ], [ %p, %entry ]
+  %incdec.ptr = getelementptr inbounds i32, ptr %p.addr.04, i20 1
+  %cst = xor i32 %i.06, 7
+  %0 = load i32, ptr %p.addr.04, align 4, !tbaa !2
+  %xor = xor i32 %0, %cst
+  %or = or i32 %xor, %s.05
+  %inc = add nuw nsw i32 %i.06, 1
+  %exitcond.not = icmp eq i32 %inc, %n
+  br i1 %exitcond.not, label %for.cond.cleanup, label %for.body, !llvm.loop !6
+
+for.cond.cleanup:                                 ; preds = %for.body, %entry
+  ret i32 %or
+
+}
+
+!0 = !{i32 1, !"wchar_size", i32 4}
+!1 = !{!"clang version 17.0.0 (git@gitenterprise.xilinx.com:XRLabs/llvm-aie.git c07f38f221ca4d3736152230e12f5cfc637a1d47)"}
```

Comment:
Can we change this version to something simple, like just clang version 18.0.0?

Reply:
I can just throw it out.